<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>kbvDBSearchClass</class>
 <widget class="QDialog" name="kbvDBSearchClass">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>261</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <property name="toolTip">
   <string notr="true"/>
  </property>
  <property name="statusTip">
   <string notr="true"/>
  </property>
  <property name="whatsThis">
   <string notr="true"/>
  </property>
  <property name="accessibleName">
   <string notr="true"/>
  </property>
  <property name="accessibleDescription">
   <string notr="true"/>
  </property>
  <property name="windowFilePath">
   <string notr="true"/>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>8</y>
     <width>451</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>1</number>
    </property>
    <item>
     <widget class="QTabWidget" name="searchTabs">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>449</width>
        <height>201</height>
       </size>
      </property>
      <property name="toolTip">
       <string notr="true"/>
      </property>
      <property name="statusTip">
       <string notr="true"/>
      </property>
      <property name="whatsThis">
       <string notr="true"/>
      </property>
      <property name="accessibleName">
       <string notr="true"/>
      </property>
      <property name="accessibleDescription">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabKeyword">
       <attribute name="title">
        <string notr="true">Terms</string>
       </attribute>
       <widget class="QLineEdit" name="searchKeywords">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>271</width>
          <height>91</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>10</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>100</height>
         </size>
        </property>
        <property name="toolTip">
         <string notr="true"/>
        </property>
        <property name="toolTipDuration">
         <number>-2</number>
        </property>
        <property name="statusTip">
         <string notr="true"/>
        </property>
        <property name="whatsThis">
         <string notr="true"/>
        </property>
        <property name="accessibleName">
         <string notr="true"/>
        </property>
        <property name="accessibleDescription">
         <string notr="true"/>
        </property>
        <property name="inputMask">
         <string notr="true"/>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="placeholderText">
         <string notr="true"/>
        </property>
       </widget>
       <widget class="QLabel" name="lblKeywords">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>200</width>
          <height>19</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>25</height>
         </size>
        </property>
        <property name="toolTip">
         <string notr="true"/>
        </property>
        <property name="statusTip">
         <string notr="true"/>
        </property>
        <property name="whatsThis">
         <string notr="true"/>
        </property>
        <property name="accessibleName">
         <string notr="true"/>
        </property>
        <property name="accessibleDescription">
         <string notr="true"/>
        </property>
        <property name="text">
         <string notr="true">Search terms</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="buttonAND">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>50</y>
          <width>121</width>
          <height>24</height>
         </rect>
        </property>
        <property name="toolTip">
         <string notr="true"/>
        </property>
        <property name="statusTip">
         <string notr="true"/>
        </property>
        <property name="whatsThis">
         <string notr="true"/>
        </property>
        <property name="accessibleName">
         <string notr="true"/>
        </property>
        <property name="accessibleDescription">
         <string notr="true"/>
        </property>
        <property name="text">
         <string notr="true">AND</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">keywordLogic</string>
        </attribute>
       </widget>
       <widget class="QRadioButton" name="buttonOR">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>30</y>
          <width>121</width>
          <height>24</height>
         </rect>
        </property>
        <property name="toolTip">
         <string notr="true"/>
        </property>
        <property name="statusTip">
         <string notr="true"/>
        </property>
        <property name="whatsThis">
         <string notr="true"/>
        </property>
        <property name="accessibleName">
         <string notr="true"/>
        </property>
        <property name="accessibleDescription">
         <string notr="true"/>
        </property>
        <property name="text">
         <string notr="true">OR</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">keywordLogic</string>
        </attribute>
       </widget>
       <widget class="QRadioButton" name="buttonAccurate">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>100</y>
          <width>131</width>
          <height>24</height>
         </rect>
        </property>
        <property name="toolTip">
         <string notr="true"/>
        </property>
        <property name="statusTip">
         <string notr="true"/>
        </property>
        <property name="whatsThis">
         <string notr="true"/>
        </property>
        <property name="accessibleName">
         <string notr="true"/>
        </property>
        <property name="accessibleDescription">
         <string notr="true"/>
        </property>
        <property name="text">
         <string notr="true">accurate</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="buttonInaccurate">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>80</y>
          <width>131</width>
          <height>24</height>
         </rect>
        </property>
        <property name="toolTip">
         <string notr="true"/>
        </property>
        <property name="statusTip">
         <string notr="true"/>
        </property>
        <property name="whatsThis">
         <string notr="true"/>
        </property>
        <property name="accessibleName">
         <string notr="true"/>
        </property>
        <property name="accessibleDescription">
         <string notr="true"/>
        </property>
        <property name="text">
         <string notr="true">inaccurate</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLabel" name="lblPaths">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>200</width>
          <height>19</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>25</height>
         </size>
        </property>
        <property name="toolTip">
         <string notr="true"/>
        </property>
        <property name="statusTip">
         <string notr="true"/>
        </property>
        <property name="whatsThis">
         <string notr="true"/>
        </property>
        <property name="accessibleName">
         <string notr="true"/>
        </property>
        <property name="accessibleDescription">
         <string notr="true"/>
        </property>
        <property name="text">
         <string notr="true">Search paths</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="searchPaths">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>150</y>
          <width>421</width>
          <height>31</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>10</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>270</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>430</width>
          <height>100</height>
         </size>
        </property>
        <property name="toolTip">
         <string notr="true"/>
        </property>
        <property name="statusTip">
         <string notr="true"/>
        </property>
        <property name="whatsThis">
         <string notr="true"/>
        </property>
        <property name="accessibleName">
         <string notr="true"/>
        </property>
        <property name="accessibleDescription">
         <string notr="true"/>
        </property>
        <property name="inputMask">
         <string notr="true"/>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="placeholderText">
         <string notr="true"/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tabDate">
       <attribute name="title">
        <string notr="true">Date</string>
       </attribute>
       <widget class="QLabel" name="lblDateFrom">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>181</width>
          <height>20</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>25</height>
         </size>
        </property>
        <property name="toolTip">
         <string notr="true" extracomment="Separate by comma"/>
        </property>
        <property name="statusTip">
         <string notr="true"/>
        </property>
        <property name="whatsThis">
         <string notr="true"/>
        </property>
        <property name="accessibleName">
         <string notr="true"/>
        </property>
        <property name="accessibleDescription">
         <string notr="true"/>
        </property>
        <property name="text">
         <string notr="true">From </string>
        </property>
       </widget>
       <widget class="QDateEdit" name="dateFrom">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>140</y>
          <width>181</width>
          <height>29</height>
         </rect>
        </property>
        <property name="toolTip">
         <string notr="true"/>
        </property>
        <property name="statusTip">
         <string notr="true"/>
        </property>
        <property name="whatsThis">
         <string notr="true"/>
        </property>
        <property name="accessibleName">
         <string notr="true"/>
        </property>
        <property name="accessibleDescription">
         <string notr="true"/>
        </property>
        <property name="minimumDateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>1752</year>
          <month>9</month>
          <day>14</day>
         </datetime>
        </property>
        <property name="maximumDate">
         <date>
          <year>2999</year>
          <month>12</month>
          <day>31</day>
         </date>
        </property>
        <property name="minimumDate">
         <date>
          <year>1752</year>
          <month>9</month>
          <day>14</day>
         </date>
        </property>
        <property name="displayFormat">
         <string notr="true">yyyy-MM-dd</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
        <property name="date">
         <date>
          <year>2099</year>
          <month>1</month>
          <day>1</day>
         </date>
        </property>
       </widget>
       <widget class="QLabel" name="lblDateTo">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>120</y>
          <width>181</width>
          <height>20</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>25</height>
         </size>
        </property>
        <property name="toolTip">
         <string notr="true" extracomment="Separate by comma"/>
        </property>
        <property name="statusTip">
         <string notr="true"/>
        </property>
        <property name="whatsThis">
         <string notr="true"/>
        </property>
        <property name="accessibleName">
         <string notr="true"/>
        </property>
        <property name="accessibleDescription">
         <string notr="true"/>
        </property>
        <property name="text">
         <string notr="true">To</string>
        </property>
       </widget>
       <widget class="QDateEdit" name="dateUntil">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>140</y>
          <width>181</width>
          <height>29</height>
         </rect>
        </property>
        <property name="toolTip">
         <string notr="true"/>
        </property>
        <property name="statusTip">
         <string notr="true"/>
        </property>
        <property name="whatsThis">
         <string notr="true"/>
        </property>
        <property name="accessibleName">
         <string notr="true"/>
        </property>
        <property name="accessibleDescription">
         <string notr="true"/>
        </property>
        <property name="maximumDate">
         <date>
          <year>2999</year>
          <month>12</month>
          <day>31</day>
         </date>
        </property>
        <property name="minimumDate">
         <date>
          <year>1752</year>
          <month>9</month>
          <day>14</day>
         </date>
        </property>
        <property name="displayFormat">
         <string notr="true">yyyy-MM-dd</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
        <property name="date">
         <date>
          <year>2099</year>
          <month>1</month>
          <day>1</day>
         </date>
        </property>
       </widget>
       <widget class="QLabel" name="lblDateInfo">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>405</width>
          <height>51</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>405</width>
          <height>100</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string notr="true"/>
        </property>
        <property name="statusTip">
         <string notr="true"/>
        </property>
        <property name="whatsThis">
         <string notr="true"/>
        </property>
        <property name="accessibleName">
         <string notr="true"/>
        </property>
        <property name="accessibleDescription">
         <string notr="true"/>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="text">
         <string notr="true">Search between two dates. FROM must be equal to / earlier than TO. Format: Year-Month-Day</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkDateSearch">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>80</y>
          <width>391</width>
          <height>24</height>
         </rect>
        </property>
        <property name="toolTip">
         <string notr="true"/>
        </property>
        <property name="statusTip">
         <string notr="true"/>
        </property>
        <property name="whatsThis">
         <string notr="true"/>
        </property>
        <property name="accessibleName">
         <string notr="true"/>
        </property>
        <property name="accessibleDescription">
         <string notr="true"/>
        </property>
        <property name="text">
         <string notr="true">Enable date search</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonAbort">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>111</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string notr="true">Abort</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonApply">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>111</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string notr="true">Apply</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>searchKeywords</tabstop>
  <tabstop>buttonOR</tabstop>
  <tabstop>buttonAND</tabstop>
  <tabstop>buttonInaccurate</tabstop>
  <tabstop>buttonAccurate</tabstop>
  <tabstop>searchPaths</tabstop>
  <tabstop>searchTabs</tabstop>
  <tabstop>checkDateSearch</tabstop>
  <tabstop>dateFrom</tabstop>
  <tabstop>dateUntil</tabstop>
  <tabstop>pushButtonAbort</tabstop>
  <tabstop>pushButtonApply</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="keywordLogic"/>
 </buttongroups>
</ui>
